src/lxml/pyclasslookup.py

```diff
@@ -1,3 +1,3 @@
-# dummy module for backwards compatibility
+"""Dummy module for backwards compatibility."""
```

Comment:
That suggests that it's not worth documenting.

Reply:
OK. Reverted and dropped from the documentation.